include/swift/AST/DiagnosticsSema.def

```diff
@@ -942,6 +942,13 @@ ERROR(need_cxx_interop_to_import_module,none,
 NOTE(enable_cxx_interop_docs,none,
      "visit https://www.swift.org/documentation/cxx-interop/project-build-setup to learn how to enable C++ interoperability", ())
 
+ERROR(cxx_stdlib_kind_mismatch,none,
+      "module %0 was built with "
+      "%select{unknown C++ stdlib|libc++|libstdc++|msvcprt}1, "
+      "but current compilation uses "
+      "%select{unknown C++ stdlib|libc++|libstdc++|msvcprt}2",
+      (Identifier, CXXStdlibKind, CXXStdlibKind))
```

Comment:
Might be nice to use a to_string to get the runtime name should we support additional ones.

Reply:
Good point, I switched this to use `to_string`.